upstairs/src/upstairs.rs

```diff
@@ -1523,7 +1542,6 @@ impl Upstairs {
                     // Any read larger than this constant should be deferred to
                     // the worker pool; smaller reads can be processed in-thread
                     // (since the overhead isn't worth it)
```

Comment:
The comment here referencing `this constant` should be updated and/or
moved to the new const location.

Reply:
Fixed in b1796e89ddf3